code/modules/clothing/outfits/ert.dm

```diff
@@ -122,7 +125,8 @@
 		/obj/item/clothing/mask/gas/sechailer/swat=1,\
 		/obj/item/gun/energy/pulse/pistol/loyalpin=1,\
```

Comment:
Medical already has sidearms as well.

Reply:
pulse is lethal only. Mini-have disablers